thirdeye/thirdeye-frontend/app/pods/components/rootcause-legend/component.js

```diff
@@ -63,7 +63,7 @@ export default Ember.Component.extend({
     'validUrns',
     function () {
       const { entities, validUrns } = this.getProperties('entities', 'validUrns');
-      return validUrns.filter(urn => entities[stripTail(urn)]).reduce((agg, urn) => { agg[urn] = entities[stripTail(urn)].color; return agg; }, {});
+      return validUrns.filter(urn => entities[urn]).reduce((agg, urn) => { agg[urn] = entities[urn].color; return agg; }, {});
```

Comment:
For readability's sake, please span over multiple lines when chaining methods.

Reply:
fixed